app/src/components/MainPage.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <PageHeader :showMenu="showMenu" :userInfo="userInfo" @toggle="toggleMenu" />
+    <MenuContent v-if="showMenu" :menuItems="menuItems" />
```

Comment:
Menu better to place inside your header component [here](https://github.com/SolomiiaLomnytska22/solomiia-training/pull/1/files#diff-9fcb333e2683dbdd27bdf792e16831a0b2b1a29c81c71cc31a885e2bd257c482R2-R13)